forch/faucet_state_collector.py

```diff
@@ -1122,6 +1123,10 @@ def process_port_learn(self, timestamp, name, port, mac, ip_addr):
             self._logger.info('Learned %s at %s:%s as %s', mac, name, port, ip_addr)
             port_attr = self._get_port_attributes(name, port)
 
+            radius_result = self.radius_results.get(mac)
```

Comment:
would this be cleaner with

mac_entry[MAC_RADIUS_RESULT] = self.radius_results.get(mac)

What is the intended behavior when there is a previous mac entry, but no radius result?   The code you have now would preserve the old (possible stale?) one?

Reply:
I could do that, but it might leave an empty 'radius_result' key.

Reply:
Added assert to account for it.